### PR TITLE
[py-backports-zoneinfo] New package

### PR DESCRIPTION
**Pull request by qwertos**

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-backports-zoneinfo/package.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    """Backport of the standard library zoneinfo module"""
+
+    homepage = "https://github.com/pganssle/zoneinfo"
+    pypi = "backports.zoneinfo/backports.zoneinfo-0.2.1.tar.gz"
+
+    version("0.2.1", sha256="fadbfe37f74051d024037f223b8e001611eac868b5c5b06144ef4d8b799862f2")
+
+    depends_on("python@3.6:", type=("build", "run"))
+    depends_on("py-setuptools@40.8.0:", type="build")
+    depends_on("py-importlib-resources", type=("build", "run"), when="^python@:3.6")
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-backports-zoneinfo/package.py]: Spack does not support Python 3.6 and older, so you can just remove this line. You could also remove the `python@3.6:` constraint as well.

File: var/spack/repos/builtin/packages/py-backports-zoneinfo/package.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+class PyBackportsZoneinfo(PythonPackage):
+    """Backport of the standard library zoneinfo module"""
+
+    homepage = "https://github.com/pganssle/zoneinfo"
+    pypi = "backports.zoneinfo/backports.zoneinfo-0.2.1.tar.gz"
+
+    version("0.2.1", sha256="fadbfe37f74051d024037f223b8e001611eac868b5c5b06144ef4d8b799862f2")
+
+    depends_on("py-setuptools@40.8.0:", type="build")
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-backports-zoneinfo/package.py:17]: Missing importlib-resources dep

**Review by adamjstewart — CHANGES_REQUESTED** [review]

File: var/spack/repos/builtin/packages/py-backports-zoneinfo/package.py (hunk 2 above)
Comment (qwertos) [reply var/spack/repos/builtin/packages/py-backports-zoneinfo/package.py:17]: In the last thread you had me remove that line?

Comment (adamjstewart) [reply var/spack/repos/builtin/packages/py-backports-zoneinfo/package.py:17]: Oops, you're right. My bad

**Review by adamjstewart — APPROVED** [review]

**Pull request merged**